Amazon.IonObjectMapper/IonObjectSerializer.cs

```diff
@@ -175,6 +230,79 @@ public override void Serialize(IIonWriter writer, object item)
             writer.StepOut();
         }
 
+        // Deserialize the given method and return bool to indicate whether the deserialized result should be used.
+        private bool TryDeserializeMethod(MethodInfo method, IIonReader reader, IonType ionType, out object deserialized)
```

Comment:
This will always return true, unless the method is not a setter which we will throw

Reply:
Correct. This was done for the sake of symmetry with the `TryDeserializeProperty` and `TryDeserializeField` methods which can return true or false.